lib/manageiq/appliance_console/message_configuration_client.rb

```diff
@@ -37,6 +38,12 @@ def activate
         true
       end
 
+      def post_activation
+        say("Restart evmserverd if it is running...")
+        evmserverd_service = LinuxAdmin::Service.new("evmserverd")
+        evmserverd_service.restart if evmserverd_service.running?
```

Comment:
Do we need to restart the server?  It should get picked up automatically, right?

Reply:
We cache the [`MiqQueue.messaging_client`](https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_queue.rb#L39-L58) so if we switched from kafka->miq-queue, artemis->miq-queue, kafka->artemis, or artemis->kafka we would have potentially stale connections cached.  We could handle this a different way though e.g. check if the messaging_type is mismatched with an existing messaging_client.

Reply:
Maybe a job for the config activator?

Reply:
@bdunne Yes the restart of evmserverd is needed. The message_type setting is not picked up automatically.

Reply:
@bdunne I think cleaning up the potentially stale cached connections is beyond the scope of this PR, and as you mention could be addressed by the config activator in a separate PR.